### PR TITLE
use urllib3==1.24.3 to avoid url parse error

### LINKED ISSUE #1537: InvalidURL: Failed to parse: http://[10.142.0.19]:9090/api/v1/status/config
### Prerequisites

* [ ] Are you rebased to master ?
* [ ] Is it reproducible ?
* [x] Did you perform a cursory search if this issue isn't opened ?

### Versions
* **SCT**: branch-3.2

### Logs
* test_id: 558840f2-12c2-4bb7-a087-2db22d7ca4e5
* job log: https://jenkins.scylladb.com/job/scylla-3.2/job/rolling-upgrade/job/rolling-upgrade-centos7/29/console

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

```
09:23:23  ======================================================================
09:23:23  ERROR: test_rolling_upgrade (upgrade_test.UpgradeTest)
09:23:23  ----------------------------------------------------------------------
09:23:23  Traceback (most recent call last):
09:23:23    File "/sct/sdcm/tester.py", line 138, in wrapper
09:23:23      args[0].tearDown()
09:23:23    File "/sct/sdcm/tester.py", line 1496, in tearDown
09:23:23      self.finalize_test()
09:23:23    File "/sct/sdcm/tester.py", line 1385, in finalize_test
09:23:23      self.stop_resources()
09:23:23    File "/sct/sdcm/tester.py", line 1422, in stop_resources
09:23:23      coredumps=db_cluster_coredumps,
09:23:23    File "/sct/sdcm/db_stats.py", line 548, in update_test_details
09:23:23      update_data['results'] = self.get_prometheus_stats()
09:23:23    File "/sct/sdcm/db_stats.py", line 482, in get_prometheus_stats
09:23:23      prometheus_db_stats = PrometheusDBStats(host=self.monitors.nodes[0].external_address)
09:23:23    File "/sct/sdcm/db_stats.py", line 135, in __init__
09:23:23      self.config = self.get_configuration()
09:23:23    File "/sct/sdcm/db_stats.py", line 159, in get_configuration
09:23:23      result = self.request(url="http://[{0.host}]:{0.port}/api/v1/status/config".format(self))
09:23:23    File "/sct/sdcm/utils/common.py", line 93, in inner
09:23:23      return func(*args, **kwargs)
09:23:23    File "/sct/sdcm/db_stats.py", line 147, in request
09:23:23      response = requests.get(url)
09:23:23    File "/jenkins/.local/lib/python2.7/site-packages/requests/api.py", line 75, in get
09:23:23      return request('get', url, params=params, **kwargs)
09:23:23    File "/jenkins/.local/lib/python2.7/site-packages/requests/api.py", line 60, in request
09:23:23      return session.request(method=method, url=url, **kwargs)
09:23:23    File "/jenkins/.local/lib/python2.7/site-packages/requests/sessions.py", line 519, in request
09:23:23      prep = self.prepare_request(req)
09:23:23    File "/jenkins/.local/lib/python2.7/site-packages/requests/sessions.py", line 462, in prepare_request
09:23:23      hooks=merge_hooks(request.hooks, self.hooks),
09:23:23    File "/jenkins/.local/lib/python2.7/site-packages/requests/models.py", line 313, in prepare
09:23:23      self.prepare_url(url, params)
09:23:23    File "/jenkins/.local/lib/python2.7/site-packages/requests/models.py", line 381, in prepare_url
09:23:23      raise InvalidURL(*e.args)
09:23:23  InvalidURL: Failed to parse: http://[10.142.0.19]:9090/api/v1/status/config
09:23:23  
09:23:23  ----------------------------------------------------------------------
09:23:23  Ran 1 test in 1797.239s
```

### PR DESCRIPTION
**Pull request by amoskong**
New urllib3 (>= 1.25) isn't allow to use square brackets for ipv4 address.
Currently we use square brackets for both ipv4 & ipv6 address.

Fixes #1537

Signed-off-by: Amos Kong <amos@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: docker/env/version [hunk 1]
```diff
@@ -1 +1 @@
-0.54
+0.55-urllib3
```
Comment (amoskong) [inline docker/env/version]: Let's remove the spare suffix before merge.

The test of this PR might create unofficial image (0.55) on builders, if this pr is changed, or other docker change uses the 0.55 version. The docker image will not match with official version.

Comment (fruch) [reply docker/env/version]: Yes we are aware of this issue, once a PR is approved we change to a formal version. The issue get ugly when too PRs are doing changes to this version, hence the use of the name

**Comment by fruch** [comment]
is that a bug in urllib ? or intentional change ?

**Comment by amoskong** [comment]
> is that a bug in urllib ? or intentional change ?

The old urllib3 didn't has strict check as RFC3986, the new urllib3 added strict check as RFC required.
So it's intentional & correct change.

This is used to avoid the problem in SCT, but we need to fix sct to not use square brackets for ipv4 address.

You can find detail in https://github.com/scylladb/scylla-cluster-tests/issues/1537

**Comment by amoskong** [comment]
the pr-merge and unittest failed, I will build docker image locally to adjust a right version of urllib3.

**Comment by amoskong** [comment]
> the pr-merge and unittest failed, I will build docker image locally to adjust a right version of urllib3.

The unittest failed, but docker image is successfully built.  The unittest fail doesn't related with this change. Just rerun the test.

**Comment by bentsi** [comment]
@fruch that's what I told you when we merged IPv6 support, square brackets is an issue for IPv4!

**Comment by fruch** [comment]
@bentsi, you did it might be an issue. it was tested and proven working. urllib3 changing the behavior wasn't a thing we could guess

**Comment by bentsi** [comment]
@amoskong is it critical to backport to branch-3.2?

**Comment by amoskong** [comment]
> @amoskong is it critical to backport to branch-3.2?

Both 3.2 & 2019.1

**Comment by amoskong** [comment]
@bentsi can you do the backport?

**Pull request merged**